app/views/admin/loans/_questions.html.slim

```diff
@@ -1,80 +1,69 @@
 section.questionnaires
-  .text-right = link_to t('questions.manage'), admin_questions_path
+  .manage-link = link_to t('questions.manage'), admin_questions_path
 
-  / Switch at top allowing toggling between criteria and post analysis
-  .filter-switch.centered-text
-    .btn-group
-      = link_to t('loan.criteria'),
-        admin_loan_tab_path(@loan.id, filter: "criteria", tab: "questions"),
-        class: "btn btn-default #{'active' if @attrib == 'criteria'}",
-        data: {filter: "criteria"}
-      = link_to t('loan.post_analysis'),
-        admin_loan_tab_path(@loan.id, filter: "post_analysis", tab: "questions"),
-        class: "btn btn-default #{'active' if @attrib == 'post_analysis'}",
-        data: {filter: "post_analysis"}
-
-  div.questionnaire data-attrib=@attrib class=@attrib
-    //unlike other forms, we start in show view even with a new record
-    section.block class=(((@response_set.persisted? && @response_set.valid?) || @response_set.new_record? ) && !@conflict ? 'show-view' : 'edit-view')
-      //hide delete link and see if there are any complaints before removing entirely
-      / .show-actions
-      /   - if policy(@loan).update? && @response_set.persisted?
-      /     = link_to [:admin, @response_set], method: :delete,
-      /       data: { confirm: t("loan.confirm_response_deletion") }
-      /       i.fa.fa-trash.fa-large>
-      /       = t(".delete")
-
-      div.container
-        div.row
-          h2 = t("loan.#{@attrib}")
-        div.row.outline-and-questionnaire
-          div.outline.collapsed
-            div.outline-expansion-control.small.hider
-              = fa_icon("chevron-left", data: {hides: "outline-container"})
-            h5.outline-expansion-control.expander
-              = fa_icon("chevron-right", data: {expands: "outline-container"})
-            div.outline-container data-expandable="outline-container"
-              h4 = t("admin.loans.questions.outline.header")
-              = render "admin/loans/questionnaires/questionnaire_outline", questions: @root.children, depth: 0, parent: @root
-          div.questionnaire-wrapper.larger
-            = render partial: "admin/loans/questionnaires/questionnaire_form",
-                     locals: {response_set: @response_set, attrib: @attrib}
+  - if @question_sets.empty?
+    = t("questions.none_for_loan_division", division_name: @loan_division.name)
+  - else
+    - if @question_sets.size > 1
+      / Switch at top allowing toggling between question sets
+      .filter-switch.centered-text
+        .btn-group
+          - @question_sets.each do |qset|
+            = link_to t("question_sets.#{qset.kind}"),
+              admin_loan_tab_path(@loan.id, tab: "questions", qset: qset.id),
+              class: "btn btn-default #{'active' if @question_set == qset}"
```

Comment:
New here is just the ability to handle zero or one question set, and a loop over question sets for the filter switch instead of hard coding them.